doc/src/manual/arrays.md

```diff
@@ -607,7 +609,7 @@ end
 
 The compressed sparse column storage makes it easy and quick to access the elements in the column
 of a sparse matrix, whereas accessing the sparse matrix by rows is considerably slower. Operations
-such as insertion of nonzero values one at a time in the CSC structure tend to be slow. This is
+such as insertion of values one at a time in the CSC structure tend to be slow. This is
```

Comment:
Perhaps go a step further, replacing "values" with "previously unstored entries", as e.g. assignment to stored entries does not behave as described in the following sentence?